### ISSUE TITLE
Unit Testing Servlets with solely JUnit  is impractical/impossible 

### ISSUE THREAD

**Issue by TwoRice** [jmsnll/QuizBiz#2]
Any instances of request and response can only be made null in the test packages and you cannot seem to
be able to create an instance of a Servlet.
My suggestion is next Sprint we can try use Mockito alongside JUnit to allow us to build mock request objects :
https://github.com/mockito/mockito

**Comment by jmsnll** [comment]
Good idea Rory! We'll be sure to discuss this at our next scrum meeting! ☺️
